### PR TITLE
feat: Updated Start Scene from canvas implementation to HTML.

### PR DESCRIPTION
**Pull request by bernhardccodev**
# Changes
- Replaced the start scene background from Canvas to HTML implementation.
- Updated Timer component creation, originally using a background class, now points to the ID background. So it won't bug out due to the implementation of Start Scene HTML background.
- Updated base button component by adding a _destroy method.
- Added test cases for background component and updated test case of base button by adding the destroy test scenario.

# How to test
- Run FTM app on browser.
- In start scene, press F12 to check if the rendered Play Button and Background are HTML implementation by viewing them on the DOM tree.
- Press Play and select any game level.
- In the game level, there should be a timer rendered.

Ref: [FM-288](https://curiouslearning.atlassian.net/browse/FM-288)

[FM-288]: https://curiouslearning.atlassian.net/browse/FM-288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by woodsjohn-codev — APPROVED** [review]

File: src/components/buttons/base-button-component/base-button-component.ts [hunk 1]
```diff
@@ -90,4 +91,9 @@ export class BaseButtonComponent {
   public getElement(): HTMLElement {
     return this.element;
   }
+
+  public _destroy() {
```
Comment (miguelccodev) [inline src/components/buttons/base-button-component/base-button-component.ts:95]: rename to `destroy()`.

Let's keep this consistent, eventually all html components will extend the base html one.

File: src/scenes/start-scene.ts [hunk 2]
```diff
@@ -158,4 +163,17 @@ export class StartScene {
     this.pwa_install_status = event;
     localStorage.setItem(PWAInstallStatus, "false");
   };
+
+  /*Note: This was from PlayButton canvas class onclick method.*/
+  private logTappedStartFirebaseEvent() {
```
Comment (miguelccodev) [inline src/scenes/start-scene.ts:168]: Add a unit test that expects `sendTappedStartEvent` to have been called.

**Review by miguelccodev — CHANGES_REQUESTED** [review]
Added comments.

File: src/components/buttons/play-button/play-button.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+@media (min-width: 768px) {
+  #play-button {
+    max-width: 200px;
+    max-height: 200px;
+  }
+}
+
+//lg
+@media (min-width: 992px) {
+
```
Comment (ashwinnair-chimple) [inline src/components/buttons/play-button/play-button.scss:33]: remove all the unused media queries to keep the code clean

File: src/components/buttons/play-button/play-button.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
+.game-scene {
+  #play-button {
+      position: absolute;
+      top: 23vh;
+      left: 50%;
+      right: 50%;
```
Comment (ashwinnair-chimple) [inline src/components/buttons/play-button/play-button.scss:6]: Only left: 50% is needed when using transform: translate(-50%, -50%)
right: 50% can be remobed

**Review by ashwinnair-chimple — COMMENTED** [review]
added comments

**Review by miguelccodev — APPROVED** [review]

**Review by ashwinnair-chimple — APPROVED** [review]
Approved

**Pull request merged**